packages/ma-components/local/page-footer/page-footer.css

```diff
@@ -0,0 +1,99 @@
+.ma-page-footer {
+  --nl-paragraph-color: currentColor;
+  --nl-heading-level-4-color: currentColor;
+  --nl-link-color: currentColor;
+  --nl-link-text-decoration-color: currentColor;
+  --nl-link-hover-color: var(--ma-page-footer-color);
+
+  background-color: var(--ma-page-footer-background-color);
+  box-sizing: border-box;
+  color: var(--ma-page-footer-color);
+  container-name: page-footer;
+  container-type: inline-size;
+  display: flex;
+  flex-direction: column;
+  inline-size: 100%;
+}
+
+.ma-page-footer__content {
+  box-sizing: border-box;
+  column-gap: var(--ma-page-footer-content-large-vw-column-gap);
+  display: flex;
+  flex-direction: column;
+  inline-size: 100%;
+  margin-inline: auto;
+  max-inline-size: var(--ma-page-footer-content-max-inline-size);
+  padding-block-end: var(--ma-page-footer-content-padding-block-end);
+  padding-block-start: var(--ma-page-footer-content-padding-block-start);
+  padding-inline-end: var(--ma-page-footer-content-padding-inline-end);
+  padding-inline-start: var(--ma-page-footer-content-padding-inline-start);
+  row-gap: var(--ma-page-footer-content-row-gap);
+
+  @container page-footer (width >= 830px) {
+    flex-direction: row;
+    padding-block-end: var(--ma-page-footer-content-large-vw-padding-block-end);
+    padding-block-start: var(--ma-page-footer-content-large-vw-padding-block-start);
+    padding-inline-end: var(--ma-page-footer-content-large-vw-padding-inline-end);
+    padding-inline-start: var(--ma-page-footer-content-large-vw-padding-inline-start);
+  }
+}
+
+.ma-page-footer__section {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  row-gap: var(--ma-page-footer-section-row-gap);
+}
+
+.ma-page-footer__section a {
+  text-decoration: none;
```

Comment:
Dit kan vast ook met `--nl-link-...` something something, zonder `a`, zou dat niet handig zijn?

Reply:
Het zijn niet alleen maar `--nl-link`'s die in de sections gebruikt worden. Ook de `utrecht-link-list__link` wordt gebruikt. Dit leek mij het meest handige. Heb je een alternatief?